tests/unit/s2n_ktls_test_utils_test.c

```diff
@@ -155,17 +156,15 @@ int main(int argc, char **argv)
                     s2n_ktls_io_stuffer_pair_free);
             EXPECT_OK(s2n_test_init_ktls_io_stuffer(server, client, &io_pair));
 
-            uint8_t count = 5;
             size_t total_sent = 0;
-            struct iovec *send_msg_iov = NULL;
-            send_msg_iov = malloc(sizeof(*send_msg_iov) * count);
-            for (size_t i = 0; i < count; i++) {
+            struct iovec send_msg_iov[sizeof(struct iovec) * S2N_TEST_MSG_IOVLEN] = { 0 };
```

Comment:
I think too many iovecs are being allocated:
```suggestion
            struct iovec send_msg_iov[S2N_TEST_MSG_IOVLEN] = { 0 };
```

Reply:
This is correct but the variable should prob be called S2N_TEST_MSG_IOVLEN_COUNT.

We are allocating enough space for 5 `struct iovec` objects.

Reply:
With this syntax the size of the array type is assumed though. If you say `struct iovec send_msg_iov[5]`, the array will be able to fit 5 iovecs.